src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,44 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+  if (typeof text === "string") {
```

Comment:
Learn about `Early Return`, wrapping your whole program in one big `if...else` like this makes it harder to follow.